docs/docs/migration.md

```diff
@@ -17,12 +17,12 @@ You need to update your `mta.yaml` file to exclude `html5` modules from the resu
       supported-platforms: []
       build-result: dist
 ```
-&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; For more information about the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms).
+For more information about the `supported-platforms` build parameter, see [Configuring and Packaging Modules According to Target Platforms](configuration.md#configuring-and-packaging-modules-according-to-target-platforms).
 
 
 * The following `build-parameters` are not supported by the Cloud MTA Build Tool: <ul><li>`npm-opts`<li>`grunt-opt`<li>`maven-opts`</ul>
 
-  If you need to change the default build behavior defined for the corresponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder). For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
+If you need to change the default build behavior defined for the corresponding builder, see [configure `custom` builder](configuration.md#configuring-the-custom-builder). For a complete list of available builders and their default behaviors, see [Builders execution commands](https://github.com/SAP/cloud-mta-build-tool/blob/master/configs/builder_type_cfg.yaml).
```

Comment:
According to the rich diff it looks like removing the space here (maybe in the previous paragraph too) will make this comment be less indented than the bullet point it should belong to, I'm not sure if it's really like this but I also don't think this is what you want to happen:
![image](https://user-images.githubusercontent.com/13255544/69914885-d237cc00-1451-11ea-8629-dc8998e58adf.png)

Reply:
fixed (I hope)